server/src/main/java/org/opensearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -149,6 +149,7 @@
     private final boolean syncGlobalCheckpointAfterOperation;
     private volatile TimeValue initialRetryBackoffBound;
     private volatile TimeValue retryTimeout;
+    private final boolean forceReplicationIfRemoteTranslogEnabled;
```

Comment:
Let's rename generically so that this could be leveraged for other use cases as well

Reply:
Ack.

Reply:
Done @Bukhtawar